File: projects/m1/008-dog-years/solutions/DomenicoTraina/dog-years.py
dog_years= int(input("Enter dog years: "))

if (dog_years<0):
    print("Please enter a value > 0")
if (0<dog_years<=2):
    human_years=dog_years*5.25
    print("{} dog years are {} human years".format(dog_years,human_years))
if (dog_years>2):
    human_years=10.5+(dog_years-2)*4
    print("{} dog years are {} human years".format(dog_years,human_years))

File: projects/m1/008-dog-years/solutions/stefandifede/dog-years.js
const dogYears = prompt("Insert dog years")

//converting user input into human age consdering: -first two years = 10.5; -each other year = 4
function conversion () {
//if user input is less than 0, display an error
    if (dogYears < 0) {
        alert("please insert a valid value for dog's years")
    }
//calculating dog years in human years 
    else {if (dogYears <= 2) {
        let convertedYears = dogYears * 10.5
        alert("Dog age converted in human years is: " + convertedYears)
    } else {
        convertedYears = (2 * 10.5) + ((dogYears - 2) * 4)
        alert("Dog age converted in human years is: " + convertedYears)
    }}
    
}

conversion()